packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -1313,11 +1313,10 @@ class _HybridAndroidViewControllerInternals extends _AndroidViewControllerIntern
   }
 }
 
-/// Controls an iOS UIView.
-///
-/// Typically created with [PlatformViewsService.initUiKitView].
-class UiKitViewController {
-  UiKitViewController._(
+/// Encapsulates common aspects of platform view controllers.
```

Comment:
Perhaps something along the lines of:
```suggestion
/// Base class for iOS and macOS view controllers.
///
/// View controllers are used to create and interact with the UIView or NSView
/// underlying a platform view.
```